L1Trigger/L1TTwinMux/test/runMuonBarrel.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import FWCore.ParameterSet.Config as cms
+process = cms.Process("L1TMuonEmulation")
+import os
+import sys
+import commands
+
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+
+process.MessageLogger.cerr.FwkReport.reportEvery = cms.untracked.int32(1)
+process.options = cms.untracked.PSet(wantSummary = cms.untracked.bool(False))
+
+process.source = cms.Source('PoolSource',
+ fileNames = cms.untracked.vstring('file:/afs/cern.ch/work/g/gkaratha/private/bmtf/gen_samples/Singlemu_oneoverpt_100k.root')
+# ,eventsToProcess=cms.untracked.VEventRange('1:1:969-1:1:969')
+
+	                    )
+
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(2400))
+
+# PostLS1 geometry used
+process.load('Configuration.Geometry.GeometryExtended2015Reco_cff')
```

Comment:
use the DB geometry

Reply:
this is a test congig file for the developer.  not used anywhere.